tests/e2e_tests/cypress/e2e/basic-test/navigation-test.cy.js

```diff
@@ -31,21 +31,23 @@ describe('Navigation Bar Test for small devices', () => {
       cy.get('.navbar-toggler').first().click();
       cy.get('#left-panel').should('be.visible');
 
-      cy.get('#filter-form').should('be.visible').then((filterForm) => {
-        const filterFormRect = filterForm[0].getBoundingClientRect();
-        const filterFormTop = filterFormRect.top;
+      cy.get('#filter-form')
+        .should('be.visible')
+        .then(filterForm => {
+          const filterFormRect = filterForm[0].getBoundingClientRect();
+          const filterFormTop = filterFormRect.top;
 
-        cy.log('Filter Form Top:', filterFormTop);
+          cy.log('Filter Form Top:', filterFormTop);
 
-        cy.get('.navbar').then(($navbar) => {
-          const navbarPosition = $navbar.position();
-          const navbarHeight = $navbar.outerHeight();
-          const navbarBottom = navbarPosition.top + navbarHeight;
-          cy.log('Navbar Bottom:', navbarBottom);
+          cy.get('.navbar').then($navbar => {
+            const navbarPosition = $navbar.position();
+            const navbarHeight = $navbar.outerHeight();
+            const navbarBottom = navbarPosition.top + navbarHeight;
+            cy.log('Navbar Bottom:', navbarBottom);
 
-          expect(filterFormTop).to.be.greaterThan(navbarBottom);
+            expect(filterFormTop).to.be.greaterThan(navbarBottom);
+          });
```

Comment:
@Dawid-Sroka - it's for sure not for this PR, but - do you think it's good approach? Maybe we should try that in future?

Reply:
This is not in the scope of this PR

Reply:
@Dawid-Sroka  - sure. I'm asking if we should create Issue out of this or ignore this comment?

Reply:
What he showed doesn't improve much. It's just extracting to cypress command instead of function.
But the term `cypress-image-snapshot` is probably worth noting down for us cypress noobs.